src/utils/command_interface.py

```diff
@@ -210,6 +214,53 @@ def command_user(self, discord_id: str) -> str:
             return_message = user["leetcode_id"]
         return return_message
 
+    def command_group_status(self) -> str:
+        """
+        Calculates and summarizes the number of users who have completed each question
+        in the current challenge
+        """
+        result = ""
+        users = self.database.table_leetcodeuser_loadall()
+        challenge = self.database.table_weeklychallenge_getlatest()
+        if not challenge:
+            result += "No current challenge"
+        elif len(users) == 0:
+            result += "No registered users"
+        else:
+            date = datetime.fromtimestamp(challenge["date"])
+            result += f"Challenge {challenge['id']} | {date.strftime('%Y-%m-%d')}\n"
+            questions = self.database.table_weeklyquestion_load_by_challenge_id(
+                challenge["id"]
+            )
+            total_questions = len(questions)
+            if total_questions == 0:
+                result += "Current challenge is empty"
+            else:
+                total_completions = 0
+                for question in questions:
+                    title = question["title"]
+                    completions = len(
+                        [
+                            user
+                            for user in users
+                            if LeetcodeUtil.check_challenge_completion(
+                                user["leetcode_id"], question["title_slug"]
+                            )
+                        ]
+                    )
+                    total_completions += completions
+                    result += f"{title}\n\t{completions}/{len(users)} users completed\n"
+                group_percentage = int(
+                    (
+                        (total_completions / (len(users) * total_questions))
+                        if total_completions > 0
+                        else 0
```

Comment:
The if/else here is should protect against divide-by-zero errors. There is nothing wrong with `total_completions` being 0 because it is in the numerator. What we don't want is `len(users) * total_questions` to be zero, more more specifically, both `users` and `total_questions` should be non-zero.

Reply:
That's right, that's left over from when I had the division flipped around. I'll revise, thanks